src/main/java/org/commcare/suite/model/RemoteQueryDatum.java

```diff
@@ -102,7 +108,7 @@ public void readExternal(DataInputStream in, PrototypeFactory pf)
         description = (Text) ExtUtil.read(in, new ExtWrapNullable(Text.class), pf);
         useCaseTemplate = ExtUtil.readBool(in);
         defaultSearch = ExtUtil.readBool(in);
-        
+        dynamicSearch = ExtUtil.readBool(in);
```

Comment:
you need to add the corresponding `write` in `writeExternal` as well

Reply:
[d35d47f](https://github.com/dimagi/commcare-core/pull/1358/commits/d35d47f54e54849a44d0a7341ee5b1568e97b901)